lib/board.rb

```diff
@@ -0,0 +1,23 @@
+# require 'game'
+class Board
+  def initialize
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+  end
+
+  def board_string
+
+    expected_output  = " #{@board[0]} | #{@board[1]} | #{@board[2]} \n"
```

Comment:
Why are you creating a variable called `expected_output` (it doesn't look like it's used)?  Is that so you could write a multiline string easily? I think the body of this function can be

```ruby
" #{@board[0]} | #{@board[1]} | #{@board[2]} \n"
+ "-----------\n"
+ " #{@board[3]} | #{@board[4]} | #{@board[5]} \n"
+ "-----------\n"
+ " #{@board[6]} | #{@board[7]} | #{@board[8]} \n"
```

and there are probably other ways to do it too.

Reply:
The variable was used because I was struggling with using HereDoc and the linter removing what it thought was trailing whitespace and it wasn't allowing me to move forward so I borrowed the code from my test. Since ruby implicitly returns the variable, the function was just returning the string. I have since played with Heredoc more and amended the test outputs to remove the third space in indexes 2, 4, and 8.